scripts/workflow-metrics-dashboard.js

```
@@ -0,0 +1,825 @@
+switch (command) {
+  case 'init':
+    await dashboard.init();
+    break;
+  case 'dashboard':
+    dashboard.displayDashboard();
+    break;
+  case 'report':
+    const reportType = args[1] || 'full';
+    dashboard.generateReport(reportType);
+    break;
+  case 'trends':
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)